src/app_service/service/community/service.nim

```diff
@@ -688,6 +688,7 @@ QtObject:
 
       updatedCommunity.settings = communitySettings
       self.allCommunities[communityId] = updatedCommunity
+      self.chatService.loadChats()
```

Comment:
One improvement we could do later is add a new function on status-go that is something like `getChatsById` so that we only get the data we need instead of everything, what do you think?